imports/plugins/core/layout/client/templates/layout/wallet/wallet.js

```diff
@@ -0,0 +1,144 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Reaction } from "/client/api";
+import { Accounts, Packages, Wallets } from "/lib/collections";
+
+Template.wallet.onCreated(function bodyOnCreated() {
+  this.state = new ReactiveDict();
+  this.autorun(() => {
+    this.state.setDefault({
```

Comment:
Setting your default state within the autorun() would lead to an unnecessary continuous resetting of the default state when a reactive state changes.